src/mlpack/core/cv/metrics/r2_score.hpp

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file r2_score.hpp
+ * @author Bisakh Mondal
+ *
+ * The R^2 (Coefficient of determination) regression metric.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_CV_METRICS_R2SCORE_HPP
+#define MLPACK_CORE_CV_METRICS_R2SCORE_HPP
+
+#include <mlpack/core.hpp>
+
+namespace mlpack {
+namespace cv {
+
+/**
+ * The R2Score is a metric of performance for regression algorithms
+ * that represents the proportion of variance (of y) that has been
```

Comment:
What do you think,  Instead of `of variance (of y)` we can add `of variance (here y)` or `of variance (y)`

Reply:
👌 Done.